README.md

```diff
@@ -157,6 +157,7 @@ The properties of the [`Box`](#box) component are also available.
 | `lastName`| `string`| - | - | If defined only the first letter will be taken and shown |
 | `src`  | `string`| - | - |  URL or base64 encoded image to be displayed in the avatar |
 | `emphasized`  | `boolean` | false | - | If true, uses a larger size avatar |
+| `background`  | `string` | "#b7bed3" | - | If defined will overwrite the default background color |
```

Comment:
Again, I don't think we need to expose this, at least to start with. Let's keep it quartenary dark for now, and if it becomes a problem we can allow to override the background.

Reply:
Ok got it @sradevski, this was a proposal for the dashboard use case and also because the avatar is placed on different colours depends on the platform. By the way I'll remove it and in case we'll add it as soon as someone needs it

Reply:
I think the color you chose is pretty good and will apply well to all use-cases, regardless what the dashboard uses now. If necessary we can always change it :)